experiments/quickcell/quickcell_vs_automatic_exhaustive_2_input.cpp

```diff
@@ -30,33 +30,55 @@ using namespace fiction;
-        std::vector<tt>{create_and_tt()}, std::vector<tt>{create_nand_tt()}, std::vector<tt>{create_or_tt()},
-        std::vector<tt>{create_nor_tt()}, std::vector<tt>{create_xor_tt()},  std::vector<tt>{create_xnor_tt()},
-        std::vector<tt>{create_lt_tt()},  std::vector<tt>{create_gt_tt()},   std::vector<tt>{create_le_tt()},
-        std::vector<tt>{create_ge_tt()},  create_crossing_wire_tt(),         create_half_adder_tt(),
-        create_double_wire_tt()};
-
-    static const std::vector<std::string> gate_names = {"and", "nand", "or", "nor", "xor", "xnor",     "lt",
-                                                        "gt",  "le",   "ge", "cx",  "ha",  "hourglass"};
+    experiments::experiment<std::string, uint64_t, double, uint64_t, uint64_t, double, double, uint64_t, double,
+                            uint64_t, double, uint64_t, double>
+        simulation_exp{"benchmark",
+                       "gate",                                 // std::string
+                       "#Total Layouts",                       // uint64_t
+                       "runtime (Automatic Exhaustive) [s]",   // double
+                       "#Gates (Automatic Exhaustive, sota)",  // uint64_t
+                       "#Gates (QuickCell)",                   // uint64_t
+                       "runtime (QuickCell, proposed) [s]",    // double
+                       "t_sota / t_proposed",                  // double
+                       "#Lp1",                                 // uint64_t
+                       "#Lp1/N [%]",                           // double
+                       "#Lp2",                                 // uint64_t
+                       "#Lp2/N [%]",                           // double
+                       "#Lp3",                                 // uint64_t
+                       "#Lp3/N [%]"};                          // double
+
+    const auto truth_tables =
+        std::vector<std::vector<tt>>{std::vector<tt>{create_id_tt()},  std::vector<tt>{create_not_tt()},
+                                     std::vector<tt>{create_and_tt()}, std::vector<tt>{create_nand_tt()},
+                                     std::vector<tt>{create_or_tt()},  std::vector<tt>{create_nor_tt()},
+                                     std::vector<tt>{create_xor_tt()}, std::vector<tt>{create_xnor_tt()},
+                                     std::vector<tt>{create_lt_tt()},  std::vector<tt>{create_gt_tt()},
+                                     std::vector<tt>{create_le_tt()},  std::vector<tt>{create_ge_tt()},
+                                     create_crossing_wire_tt(),        create_half_adder_tt(),
+                                     create_double_wire_tt()};
+
+    static const std::vector<std::string> gate_names = {"inv", "wire", "and", "nand", "or", "nor", "xor",      "xnor",
+                                                        "lt",  "gt",   "le",  "ge",   "cx", "ha",  "hourglass"};
```

Comment:
You could merge the two into a `std::array<std::pair<std::vector<tt>, std::string>>` data type.

Reply:
Analogously in the `quickcell_3_input.cpp`